packages/sdk/src/RequestController.ts

```diff
@@ -39,6 +45,21 @@ export class RequestController {
 		})
 	}
 
+	public getApiLimits(): ApiLimits {
+		return this.apiLimits
+	}
+	private setApiLimits(headers: Record<string, any>) {
+		const rateLimit = headers['x-rate-limit-limit'] ?? false
```

Comment:
Redo it to use `lastResponseHeaders` variable as described above and drop passing headers via argument